CHANGELOG.md

```diff
@@ -3,6 +3,11 @@
 
 ## vN.N.N
 * issue #1035: Add Spotify Embed Block
```

Comment:
@jalopez13 this ticket # is from the scottish repo, please remove it

Reply:
@efjacobson 

Removed